src/Ng/Models/RegistrationItem.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using Newtonsoft.Json;
+using Catalog = NuGet.Services.Metadata.Catalog;
+
+namespace Ng.Models
+{
+    /// <summary>
+    /// Model for a NuGet registration item, https://api.nuget.org/v3/registration1/newtonsoft.json/8.0.2.json.
+    /// </summary>
+    public class RegistrationItem
```

Comment:
Umm, yes, this file was part of the last PR. I was supposed to delete it. I'm not sure if I merged incorrectly, but I don't know how this file made it back in. I will attempt to remove it again. Sorry about that.